pubspec.yaml

```diff
@@ -30,6 +30,8 @@ dependencies:
   shared_preferences: ^0.5.6+3
   # Common Fonts
   google_fonts: ^0.5.0+1
+  # Upload Image
```

Comment:
Add good comments
image_picker is a package to pick an image from gallery or click from camera
its not for uploading image
refer the usual one-liner on [pub.dev](https://pub.dev) for better plugin doc